spec/models/guest_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Guest, type: :model do
+ it { should have_many :rooms }
+
+  describe ".scope" do 
+    it '.count_guests' do 
```

Comment:
```ruby
describe "Scopes" do
  describe "::count_guests" do
    it "should return the total number of guests" do
```

The `it` block should explain what the method should do. Use the `describe` block to name what method is being tested.